scripts/mirador.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+echo "Installing Mirador"
+
+#HOME_DIR=$1
+#if [ -f "$HOME_DIR/islandora/configs/variables" ]; then
+#  . "$HOME_DIR"/islandora/configs/variables
+#fi
+#cd "$HOME_DIR"
+
+cd /tmp && { curl -LOk https://github.com/NVLI/mirador-js/archive/master.zip; curl -LOk https://ftp.drupal.org/files/projects/mirador-8.x-1.x-dev.zip ; cd -;}
+
+cd /var/www/html/drupal/web/libraries && { unzip /tmp/master.zip -d . ;}
```

Comment:
Here you can replace the hardcoded `/var/www/html/drupal` with `$DRUPAL_HOME` from the variables file too.